docs/art/views/full_menu_view.md

```diff
@@ -0,0 +1,231 @@
+---
```

Comment:
These docs are great!

Reply:
Thanks, once this PR is done I was thinking I'd make similar docs for the other views so it will be consistent.

Reply:
Oh man that would be great. I've put a lot of work into the docs, but as you can see they still need a lot more love!